README.md

```diff
@@ -69,14 +69,13 @@ Refer [error-handling.md](https://github.com/cruxprotocol/js-sdk/blob/master/err
         - subdomain part of [CruxID](#cruxid)
     - Returns: Promise resolving to a _boolean_ indicating whether a particular Crux ID is available for registration.
 
-2. ##### getAssetMapping()
+2. ##### getAssetMap()
```

Comment:
Lets not do breaking changes unless necessary. You can add an alias and change the documentation so the Zel's code doesn't break.